inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -700,13 +700,6 @@ V10Parser::V10Parser::GenericLayerParams XmlDeserializer::parseGenericParams(
             port.dims.push_back(dim);
         }
 
-        ngraph::element::Type type(ngraph::element::Type_t::undefined);
```

Comment:
@lazarevevgeny it means that MO shouldn't generate precisions for output ports.

Reply:
For backward compatibility reasons we keep them in IRv10. But in IRv11 we will remove it.